.github/workflows/deploy.yml

```diff
@@ -0,0 +1,38 @@
+name: Deployment
+
+on:
+  # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch
```

Comment:
> Adds GitHub action workflow to deploy the website on demand.

I think JZ asked something similar in a thread that was resolved, but I wasn't able to figure out when this workflow should be triggered. @Francois-Esquire could you provide clarification on why and when I would use this?